packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -82,6 +83,7 @@ export class TestObjectProvider<TestContainerConfigType> {
             urlResolver: this.urlResolver,
             documentServiceFactory: this.documentServiceFactory,
             codeLoader,
+            logger:ChildLogger.create(getTestLogger?.(), undefined, {driverType: this.driver.type}),
```

Comment:
```suggestion
            logger:ChildLogger.create(getTestLogger?.(), undefined /* namespace */, { driverType: this.driver.type }),
```

Reply:
I'm wondering if it would be nice or not to have `createLoader` take a string `loggerNamespace` that tests could use (e.g. pass in the test group/file name). I'll see what else you're doing to help us correlate log lines to tests...

Reply:
Ok finished the rest of the file - I think some kind of correlation between test run and test case/file/etc is important. See #5297.  Fine if it's another PR though.

Reply:
The mocha hook should do this

Reply:
i wouldn't much with the namespace as that will change all the event names. it should just be added as a default prop